flathub.json

```
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64", "i386"]
```

i386 hasn't been supproted for over THREE YEARS.

```suggestion
  "only-arches": ["x86_64"]
```

---

Oh my, somehow I let this slip... My bad : <